frontend-react/src/components/shared/Table/Table.tsx

```diff
@@ -0,0 +1,225 @@
+            ? rowData.sort((a, b): number => {
+                  let contentColA = convertStringToSortableRepresentation(
+                      a.find((item) => item.columnKey === activeColumn)?.content
+                  );
+                  let contentColB = convertStringToSortableRepresentation(
+                      b.find((item) => item.columnKey === activeColumn)?.content
+                  );
+
+                  if (sortOrder === FilterOptionsEnum.ASC) {
+                      return contentColA < contentColB ? 1 : -1;
+                  } else {
+                      return contentColA > contentColB ? 1 : -1;
+                  }
+              })
+            : rowData;
+    }, [activeColumn, rowData, sortOrder]);
+    return (
+        <>
+            <thead>
+                <tr>
+                    {columnHeaders.map((header) => {
+                        return (
+                            <SortableTableHeader
+                                columnTitle={header}
+                                activeColumn={activeColumn}
+                                sortOrder={sortOrder}
+                                setActiveColumn={setActiveColumn}
+                                setSortOrder={setSortOrder}
+                            />
+                        );
+                    })}
+                </tr>
+            </thead>
+            <tbody>
+                {sortedData.map((row) => {
+                    return (
+                        <tr>
+                            {row.map((data) => {
+                                return (
+                                    <td
+                                        dangerouslySetInnerHTML={{
+                                            __html: sanitizeHtml(data.content),
+                                        }}
+                                    />
+                                );
+                            })}
+                        </tr>
+                    );
+                })}
+            </tbody>
+        </>
+    );
+};
+
+export const Table = ({
+    borderless,
+    compact,
+    emptyState,
+    fullWidth,
+    scrollable,
+    sortable,
+    stackedStyle,
+    striped,
+    rowData,
+}: TableProps) => {
+    const classes = classnames("usa-table", {
+        "usa-table--borderless": borderless,
+        "usa-table--compact": compact,
+        "width-full": fullWidth,
+        "usa-table--stacked": stackedStyle === "default",
+        "usa-table--stacked-header": stackedStyle === "headers",
+        "usa-table--striped": striped,
+    });
+
+    const columnHeaders = [
+        ...new Set(rowData.flat().map((data: RowData) => data.columnKey)),
+    ];
+
+    return (
+        <div
+            className={styles.reportstreamTable}
+            {...(scrollable && {
+                className: `usa-table-container--scrollable ${styles.reportstreamTable}`,
+                tabIndex: 0,
```

Comment:
Why does this need to be tabbable if it's scrollable?  In any case, it's not great practice to add a tab index to non-interactive elements like `<div>`s.

Reply:
It's in the USWDS docs: https://designsystem.digital.gov/components/table/#accessibility-table

Check the section on scrollable tables.